muted-tests.yml

```diff
@@ -156,15 +153,9 @@ tests:
 - class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
   method: test {p0=cat.shards/10_basic/Help}
   issue: https://github.com/elastic/elasticsearch/issues/116110
-- class: org.elasticsearch.xpack.restart.CoreFullClusterRestartIT
-  method: testSnapshotRestore {cluster=UPGRADED}
-  issue: https://github.com/elastic/elasticsearch/issues/111799
-- class: org.elasticsearch.xpack.restart.CoreFullClusterRestartIT
-  method: testSnapshotRestore {cluster=OLD}
-  issue: https://github.com/elastic/elasticsearch/issues/111774
-- class: org.elasticsearch.upgrades.FullClusterRestartIT
-  method: testSnapshotRestore {cluster=OLD}
-  issue: https://github.com/elastic/elasticsearch/issues/111777
+- class: org.elasticsearch.xpack.spatial.search.GeoGridAggAndQueryConsistencyIT
+  method: testGeoPointGeoTile
+  issue: https://github.com/elastic/elasticsearch/issues/115818
```

Comment:
This also seems unrelated - should this test be muted in `main` instead?

Reply:
Must be a fallout from resolving merge conflict. Removed in 0713e01456b05a3e5df96f99c47a0d289408a5cf

Reply:
@DaveCTurner you are correct. It's updated by our release automation.

https://github.com/elastic/elasticsearch/commit/56bc6fda6f342a435837f0eef378fd96f70cab5a#diff-84af7e2f9a12f89edad6ecb30bbb1f60b45a285b2fdcec8b62d7dfc6ae64d6ee